Elevel/Elevel.Api/appsettings.json

```diff
@@ -9,7 +9,8 @@
 
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true",
+    "DefaultConnection": "Server=PC\\SQLEXPRESS;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true",
+    //"DefaultConnection": "Server=localhost;Database=Elevel;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
please do not change connection string again and again

Reply:
Ok